2000-08-09  Bryce McKinlay  <bryce@albatross.co.nz>

	* java/awt/Color.java (getTransparency): New method.

--- libgcj/libjava/java/awt/Color.java.jj	Fri Dec 15 18:28:57 2000
+++ libgcj/libjava/java/awt/Color.java	Fri Dec 15 18:29:01 2000
@@ -87,4 +87,12 @@ public class Color extends Object implem
   {
     return rgba;
   }
+  
+  public int getTransparency()
+  {
+    if (getAlpha() == 0xFF)
+      return Transparency.OPAQUE;
+    else
+      return Transparency.TRANSLUCENT;
+  }
 }
